fast/stages/0-bootstrap/outputs.tf

```diff
@@ -20,6 +20,9 @@ locals {
   cicd_workflows = {
     for k, v in local.cicd_repositories : k => templatefile(
       "${path.module}/templates/workflow-${v.type}.yaml", {
+        allowed_audiences = try(
```

Comment:
Can we move the `try` inside the locals? I'm not 100% sure the `oidc` block is always populated. If you are and it's always populated, why do we need a `try` here?